app/templates/services/previous_services.html

```diff
@@ -35,62 +35,124 @@
-          data-analytics-category="Copy services"
-          data-analytics-action="Copy all"
-          data-analytics-label="Count: {}">Yes, add all services'.format(previous_services|length)|safe
+        errors = form_errors
       %}
-        {% include "toolkit/notification-banner.html" %}
+        {% include 'toolkit/forms/validation.html' %}
       {% endwith %}
-    </form>
-  {% endif %}
+    {% endif %}
 
-  {% include "partials/service_warning.html" %}
+    {% include "partials/service_warning.html" %}
 
-  {% with
-     heading = "Previous " + lot.name|lower + " services",
-     smaller = True,
-     with_breadcrumb = True
-  %}
-    {% include "toolkit/page-heading.html" %}
-  {% endwith %}
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <div class="single-question-page">
+          {% with
+             heading = form.copy_service.label,
+             smaller = True,
+             with_breadcrumb = True
+          %}
+            {% include "toolkit/page-heading.html" %}
+          {% endwith %}
+
+          <p>You still have to:</p>
+          <ul class="list-bullet">
+            <li>review your service</li>
+            <li>answer any new questions</li>
+          </ul>
+          <br>
+
+          <form method="POST" action="{{ url_for('.previous_services', framework_slug=framework.slug, lot_slug=lot.slug) }}">
+            <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+            {%
+              with
+              name = "copy_service",
+              type = "boolean",
+              error = form.copy_service.errors[0]
+            %}
+              {% include "toolkit/forms/selection-buttons.html" %}
+            {% endwith %}
+
+            {%
+              with
+              type = "save",
+              label = "Save and continue",
+              analytics = "trackEvent",
+              analytics_category = "Copy services",
+              analytics_action = "Copy one lot service",
+              analytics_label = lot.slug
+            %}
+              {% include "toolkit/button.html" %}
+            {% endwith %}
+          </form>
+        </div>
+      </div>
+    </div>
+  {% else %}
+    {% if copy_all %}
+      <form action="{{ url_for('.copy_all_previous_services', framework_slug=framework.slug, lot_slug=lot.slug) }}" method="POST">
+        <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
+        {%
+          with
+          message = "Are you sure you want to add all your {} services?".format(lot.name|lower),
+          type = "success",
+          action = '<button type="submit" class="button-save banner-action" data-analytics="trackEvent"
+            data-analytics-category="Copy services"
+            data-analytics-action="Copy all"
+            data-analytics-label="Count: {}">Yes, add all services'.format(previous_services_still_to_copy|length)|safe
```

Comment:
Should this content match whatever we change the other label to?

Reply:
The link text you'd click to show this banner? That'll say 'Add all your services?'

Reply:
I specifically meant the analytics label being consistent with the displayed value. But maybe on second thought it should stay the same, to remain backwards compatible with past data.